app/routes/achievements/components/Leaderboard.tsx

```diff
@@ -0,0 +1,84 @@
+import { Flex, LoadingIndicator, Page } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Link } from 'react-router-dom';
+import { fetchLeaderboardUsers } from 'app/actions/AchievementActions';
+import Table from 'app/components/Table';
+import { selectUsersWithAchievementScore } from 'app/reducers/users';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { AchievementTabs } from './utils';
+import type { ColumnProps } from 'app/components/Table';
+import type { PublicUser } from 'app/store/models/User';
+
+type RankedUser = PublicUser & {
+  rank: number;
+};
+
+const Leaderboard = () => {
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchLeaderboardUsers',
+    () => {
+      dispatch(fetchLeaderboardUsers());
+    },
+    [dispatch],
+  );
+  const users = useAppSelector((state) =>
+    selectUsersWithAchievementScore(state),
+  );
+  const fetching = useAppSelector((state) => state.users.fetching);
+  const rankedUsers: RankedUser[] = users
+    .slice()
+    .sort((a, b) => b.achievementScore - a.achievementScore)
+    .map((user, index) => ({
+      ...user,
+      rank: index + 1,
+    }));
+  const columns: ColumnProps<RankedUser>[] = [
```

Comment:
```suggestion
    }));
    
  const columns: ColumnProps<RankedUser>[] = [
```
Spaces is your friend